2018/OSFC/README.md

```diff
@@ -0,0 +1,33 @@
+## Conference: Open Source Firmware Conference
+
+* **Date**: 12-15.09.2018
+* **Location**: Erlangen, Germany
+* **website**: -
+* **social media**: -
+
+### Presentation #1
+
+* **Topic**: Remote Testing Environment Workshop
+* **Presenters**: Piotr Król, Michał Żygowski
+* **slides**: https://2018.osfc.io/uploads/talk/paper/27/Remote_Testing_Environment_Workshop.pdf
+* **white papers**: -
+* **YouTube**: -
+* **Misc**: -
+
+### Presentation #2
+
+* **Topic**: How to enable AMD IOMMU in coreboot
+* **Presenters**: Piotr Król
+* **slides**: https://2018.osfc.io/uploads/talk/paper/28/How_to_enable_AMD_IOMMU_in_coreboot.pdf
+* **white papers**: -
+* **YouTube**: https://www.youtube.com/watch?time_continue=2&v=5JoEuh9qXx0I
```

Comment:
@artur-rs we also should point to other resources e.g. internal triage presentaiton, sometimes those are different:
https://www.youtube.com/watch?v=gBsHwAjgeh0&t=7s
https://www.youtube.com/watch?v=YlRvj2lqpHE&t=7s